cdk/drupal/drupal_stack.py

```diff
@@ -966,7 +976,7 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
             "CloudFrontOriginAccessPolicyParam",
             allowed_values = [ "http-only", "https-only", "match-viewer" ],
             default="match-viewer",
```

Comment:
@dylanvaughn I think we have some issues with the Cloudfront-Origin configuration that aren't quite smoothed out. I debugged some 502 gateway issues last week and now I think there is more to look into, but pending that investigation, do you think this parameter could get us into trouble? If the customer doesn't supply a CertificateARN, CloudFront would be unable to access the origin via HTTPS, so the https-only option wouldn't work...

Reply:
@acarlton I created a ticket to look into this more.